mods/ra/maps/fort-lonestar/map.yaml

```diff
@@ -23,12 +23,13 @@ Videos:
 Options:
 	Fog: True
 	Shroud: True
-	AllyBuildRadius: False
+	AllyBuildRadius: True
 	FragileAlliances: False
 	StartingCash: 50
 	TechLevel: Unrestricted
 	ConfigurableStartingUnits: False
 	ShortGame: False
+	Difficulties: Hard (4P), Normal (3P), Easy (2P), Very Easy (1P), Real tough guy
```

Comment:
People might still want an easier/harder option that the automatically-picked one.